blurring_as_a_service/cleaning_pipeline/source/smart_sampler.py

```diff
@@ -287,7 +275,7 @@ def _get_n_random_images_per_date(
     @staticmethod
     def _categorize_images_into_bins(
         df: pd.DataFrame,
-    ) -> Tuple[Dict[str, pd.DataFrame], Dict[str, pd.DataFrame]]:
+    ):
```

Comment:
there is no return type here anymore

Reply:
I know, I didn't know what was the right return type and since we are not strict with typing I removed it 😆